.ci_support/osx_.yaml

```diff
@@ -3,7 +3,7 @@ MACOSX_DEPLOYMENT_TARGET:
 c_compiler:
 - clang
 c_compiler_version:
-- '4'
+- '9'
```

Comment:
In the `defaults` channel those compilers for OSX are not available yet. Please check how we worked it around in the xraylib-feedstock.